nautobot_chatops/dispatchers/mattermost.py

```diff
@@ -574,12 +577,14 @@ def bold(self, text):
         """Mark text as bold."""
         return f"**{text}**"
 
+    # pylint: disable=arguments-differ
```

Comment:
This one worries me - what arguments are differing?

Reply:
The message was something like this:
W0221: Parameters differ from overridden 'send_blocks' method (arguments-differ)